stl/inc/format

```diff
@@ -1248,15 +1243,426 @@ _OutputIt _Write(_OutputIt _Out, basic_string_view<_CharT> _Value) {
     return _STD copy(_Value.begin(), _Value.end(), _Out);
 }
 
-// TODO: placeholder write
-template <class _CharT, class _OutputIt, class _Ty>
-_OutputIt _Write(_OutputIt _Out, _Ty _Val, _Basic_format_specs<_CharT>& _Specs) {
+template <class _CharT, class _OutputIt, class _Func>
+_OutputIt _Write_aligned(
+    _OutputIt _Out, int _Width, _Basic_format_specs<_CharT>& _Specs, _Align _Default_align, _Func&& _Fn) {
+    int _Fill_left  = 0;
+    int _Fill_right = 0;
+
+    if (_Specs._Alignment == _Align::_None)
+        _Specs._Alignment = _Default_align;
+
+    if (_Width < _Specs._Width) {

```

Comment:
What is the expectation when width is larger than what the user asked for. Should we crop the output?

Reply:
Nope, it's a minimum width.